_config.yml

```diff
@@ -48,6 +48,5 @@ exclude: ["downloads/RELEASE_NOTE.md","vendor","0.991/","v1-2","v1-1","v1-0","v0
 #   - vendor/cache/
 #   - vendor/gems/
 #   - vendor/ruby/
-dist_server: https://dist.ballerina.io
-
-
+dist_server: http://dist-dev.ballerina.io
```

Comment:
This shouldn't be changed. Production should point to `https://dist.ballerina.io`